exercises/practice/binary-search/.docs/instructions.md

```diff
@@ -33,3 +33,9 @@ found in the array and a special "not found" indication is returned.
 A binary search halves the number of items to check with each iteration,
 so locating an item (or determining its absence) takes logarithmic time.
 A binary search is a dichotomic divide and conquer search algorithm.
+
+## Restrictions
+
+Do not sell yourself short by using the standard library of collection
+tools. You can and should complete this exercise without tools like
+`Array.tryFindIndex`.
```

Comment:
Could you move this to the append file?

Reply:
This appears in the main instructions file for [accumulate](https://github.com/exercism/fsharp/blob/main/exercises/practice/accumulate/.docs/instructions.md#restrictions).

Should that be changed as well?

Reply:
No, because that is in the original metadata from which it is synced: https://github.com/exercism/problem-specifications/blob/main/exercises/accumulate/description.md

Reply:
Completed: 6a69106

I'm still a bit confused by this feedback to be honest.